addons/mail/static/src/messaging/component/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -842,6 +842,120 @@ QUnit.test('open 2 different chat windows: enough screen width', async function
     );
 });
 
+QUnit.test('open 2 chat windows: check right shift operations are available', async function (assert) {
```

Comment:
I would remove "right" (misleading because it means "correct" or "on the right-hand side")